README.md

```diff
@@ -1,19 +1,60 @@
 # .NET Source-build Reference Packages
 
-This repository contains source and build scripts for source-buildable reference versions of historical .NET Core packages that are referenced by the [source-build](https://github.com/dotnet/source-build) repo.  These are used only when building the source-build repo.
+This repository contains tools, source and build scripts for source-buildable reference
+versions of historical .NET Core packages that are referenced by
+[.NET source build](https://github.com/dotnet/source-build). These are used only
+when source building .NET.
 
-### Continuous builds
+This repo supports three package types
 
-Microsoft build outputs from this repo are published by Arcade to blob storage.
-They are used by source-build when prebuilts are acceptable to avoid rebuilding
-the repo. This repo doesn't change often, making a cache particularly useful.
+1. Reference - A package that contains API signatures, no implementation.  It enables developers to build targeting a specific library version without having the full implementation assembly for that version.
+1. Targeting - A bundle of reference packages that target a specific .NET TFM.
+1. Text only - Packages that contain text only assets.
 
-### Build on Linux or MacOS
+## Supported Platforms
 
+.NET source build currently only supports Linux.
+
+## Building
+
+``` bash
+./build.sh -sb
 ```
-./build.sh
+
+## Adding new Packages
```

Comment:
This doesn't describe why it would be necessary to add new packages. It might be worthwhile to describe the scenarios where this is done.

Reply:
Added

```
New packages are needed from time to time as
[existing dependency versions are upgraded](https://github.com/dotnet/source-build/blob/main/Documentation/sourcebuild-in-repos/update-dependencies.md)
and [new dependencies are added](https://github.com/dotnet/source-build/blob/main/Documentation/sourcebuild-in-repos/new-dependencies.md).
```